api/config/routes.rb

```diff
@@ -8,124 +8,119 @@
     end
   end
 
-  concern :api do
-    resources :promotions, only: [:show]
-
-    resources :products do
-      resources :images
-      resources :variants
-      resources :product_properties
-    end
+  namespace :api, defaults: { format: 'json' } do
+    namespace :v1 do
+      resources :promotions, only: [:show]
 
-    concern :order_routes do
-      member do
-        put :cancel
-        put :empty
-        put :apply_coupon_code
+      resources :products do
+        resources :images
+        resources :variants
+        resources :product_properties
       end
 
-      resources :line_items
-      resources :payments do
+      concern :order_routes do
         member do
-          put :authorize
-          put :capture
-          put :purchase
-          put :void
-          put :credit
+          put :cancel
+          put :empty
+          put :apply_coupon_code
         end
-      end
 
-      resources :addresses, only: [:show, :update]
+        resources :line_items
+        resources :payments do
+          member do
+            put :authorize
+            put :capture
+            put :purchase
+            put :void
+            put :credit
+          end
+        end
 
-      resources :return_authorizations do
-        member do
-          put :add
-          put :cancel
-          put :receive
+        resources :addresses, only: [:show, :update]
+
+        resources :return_authorizations do
+          member do
+            put :add
+            put :cancel
+            put :receive
+          end
         end
       end
-    end
 
-    resources :checkouts, only: [:update], concerns: :order_routes do
-      member do
-        put :next
-        put :advance
+      resources :checkouts, only: [:update], concerns: :order_routes do
+        member do
+          put :next
+          put :advance
+        end
       end
-    end
 
-    resources :variants do
-      resources :images
-    end
+      resources :variants do
+        resources :images
+      end
 
-    resources :option_types do
+      resources :option_types do
+        resources :option_values
+      end
       resources :option_values
-    end
-    resources :option_values
 
-    resources :option_values, only: :index
+      resources :option_values, only: :index
 
-    get '/orders/mine', to: 'orders#mine', as: 'my_orders'
-    get "/orders/current", to: "orders#current", as: "current_order"
+      get '/orders/mine', to: 'orders#mine', as: 'my_orders'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.